packages/jest-circus/src/utils.js

```diff
@@ -272,3 +270,31 @@ export const invariant = (condition: *, message: string) => {
     throw new Error(message);
   }
 };
+
+// See: https://github.com/facebook/jest/pull/5154
+function convertDescriptorToString(descriptor) {
```

Comment:
Hate this, but compat 🤷‍♂️

Reply:
is there a place we could put it in jest and share between both jasmine and circus?

Reply:
Could add this to `jest-util`, not sure if we want it though (requires another dep for circus)

Reply:
+ 1 dep > duplicated code 😛

Reply:
i'm fine with both though!

Reply:
fixd